kernel_hardening_checker/checks.py

```diff
@@ -663,21 +663,34 @@ def add_sysctl_checks(l, _arch):
 # Calling the SysctlCheck class constructor:
 #   SysctlCheck(reason, decision, name, expected)
 
-    l += [SysctlCheck('self_protection', 'kspp', 'net.core.bpf_jit_harden', '2')]
+    # Use an omnipresent kconfig symbol to see if we have a kconfig file for checking
+    have_config_file = KconfigCheck('-', '-', 'LOCALVERSION', 'is present')
+
+    l += [OR(SysctlCheck('self_protection', 'kspp', 'net.core.bpf_jit_harden', '2'),
+             AND(KconfigCheck('-', '-', 'BPF_JIT', 'is not set'),
+                 have_config_file))]
```

Comment:
Here was the error: KSPP doesn't recommend to disable `BPF_JIT`.

I fixed it in https://github.com/a13xp0p0v/kernel-hardening-checker/pull/125/commits/b7fd998a9cfa6c25fad3a530a15d5ac9a9bf8c83